TicTacToe/TicTacToe.cs

```diff
@@ -25,55 +26,118 @@ public static void Main()
             Console.ReadLine();
         }
 
+        // get row and column input from user
         public static void GetInput()
         {
             Console.WriteLine("Player " + playerTurn);
             Console.WriteLine("Enter Row:");
             int row = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter Column:");
             int column = int.Parse(Console.ReadLine());
+            PlaceMark(row, column);
         }
 
+
+        // place mark on the board in appropriate location, check for win, check for tie, and continue to next player move if neither
         public static void PlaceMark(int row, int column)
```

Comment:
A lot more than just placing the marker is in this method. I would split this into at least 2 other methods: CheckForEndGame and PlaceMark (you already have this method). Then call CheckForEndGame in your Program.Main after PlaceMark